crates/cxx-qt-lib/src/core/qdatetime.rs

```diff
@@ -56,10 +56,12 @@ mod ffi {
         fn offsetFromUtc(self: &QDateTime) -> i32;
```

Comment:
remaining issue appears to be with Qt 6.8 the linking order is incorrect with Qt6QuickControls2 being linked before our staticlib